main.go

```diff
@@ -10,13 +10,16 @@ import (
 	"encoding/json"
 	"flag"
 	"fmt"
-	"github.com/pelletier/go-toml"
 	"io"
 	"log"
+	"net/http"
 	"os"
 	"os/exec"
 	"path/filepath"
 	"strings"
+
+	"github.com/pelletier/go-toml"
+	"golang.org/x/net/html"
```

Comment:
Is there a particular reason why you re-organized the imports? I suggest running `gofmt -s` on the file to keep the imports in line with the existing import statements.

Reply:
my vim setup (vim-go) is running go fmt, I'll try and put it how it was